src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx

```diff
@@ -79,6 +91,8 @@ class FollowArtistPopoverRow extends React.Component<Props, State> {
               }, 500)
             }
           )
+
+          excludeState.addArtist(node._id)
```

Comment:
We add the fetched suggestion (so other rows know not to fetch that same suggestion).

Reply:
I actually was going to, but intentionally named it with `State` at the end. Since, this isn't actually the list of id's, but the actual `unstated` state object (with the accompanying `addArtist` method). For the filters, this is called `filterState`, which I think clearly makes sense in that context, so I was trying to follow that.

Maybe `excludeArtistIdsState`?

Reply:
More verbose but better 👍